src/pymovements/transforms.py

```diff
@@ -202,13 +202,15 @@ def pos2vel(
         v = v * sampling_rate
 
     else:
-        raise ValueError(f'Method needs to be in {valid_methods}'
-                         f' (is: {method})')
+        raise ValueError(
+            f'Method needs to be in {valid_methods}'
+            f' (is: {method})',
+        )
 
     return v
 
 
-def vnorm(arr: np.ndarray, axis: Optional[int] = None) -> np.ndarray:
+def vnorm(arr: np.ndarray, axis: int | None = None) -> np.ndarray:
```

Comment:
i mentioned that i prefer Optional, but now I have to say this one looks better actually